From 9f02c90c89f272598d1f2c130b12a075599e0ba7 Mon Sep 17 00:00:00 2001
From: Dimitri John Ledkov <dimitri.j.ledkov@intel.com>
Date: Thu, 11 Dec 2014 19:46:30 +0000
Subject: [PATCH] kmod: Install 'legacy' scripts.
Organization: Intel Corporation (UK) Ltd. - Co. Reg. #1134945 - Pipers Way, Swindon SN3 1RJ

In practice they are required by many projects be it for testsuites
(e.g. systemd) or at runtime (dracut), and all distributions do or
recommend creating these symlinks in sbin (e.g. Fedora, Debian, Linux
from Scratch etc.).

Signed-off-by: Dimitri John Ledkov <dimitri.j.ledkov@intel.com>
---
 Makefile.am | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index ea5cbac..1e30d7c 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -116,7 +116,7 @@ uninstall-hook:
 if BUILD_TOOLS
 bin_PROGRAMS = tools/kmod
 
-noinst_SCRIPTS = \
+sbin_SCRIPTS = \
 	tools/insmod tools/rmmod tools/lsmod \
 	tools/modprobe tools/modinfo tools/depmod
 
@@ -130,7 +130,7 @@ tools_kmod_LDADD = \
 	shared/libshared.la \
 	libkmod/libkmod-internal.la
 
-${noinst_SCRIPTS}: tools/kmod
+${sbin_SCRIPTS}: tools/kmod
 	$(AM_V_GEN) ($(RM) $@; \
 		$(LN_S) $(notdir $<) $@)
 endif
-- 
2.1.0

